devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java

```diff
@@ -210,11 +205,15 @@ private QuarkusDevModeLauncher newLauncher() throws Exception {
         final Project project = getProject();
 
         String java = null;
-        Provider<JavaLauncher> javaLauncher = getJavaLauncher();
-        if (javaLauncher.isPresent()) {
-            java = javaLauncher.get().getExecutablePath().getAsFile().getAbsolutePath();
-        }
 
+        if (toolchainSupported) {
```

Comment:
Why another flag? Isn't easier to just change the condition here?

```suggestion
        if ((GradleVersion.current().compareTo(GradleVersion.version("6.7"))) {
```

Reply:
right, it's simpler :)